.github/workflows/container-reusable.yml

```diff
@@ -0,0 +1,82 @@
+name: Refresh container image
+
+on:
+  workflow_call:
+    inputs:
+      container-tag:
+        required: true
+        type: string
+      branch:
+        required: true
+        type: string
+      base-container:
+        required: false
+        type: string
+#    secrets:
+#      token:
+#        required: true
+#      QUAY_USERNAME:
+#        required: true
+#      QUAY_PASSWORD:
+#        required: true
```

Comment:
I could find no way of verifying that these are passed with the 'inherit' keyword, so I'll leave these here until we know how to approach that:
- explicitly pass limited amount of secrets = uncomment here, add to caller
- believe it works fine, delete this block, and merge
- something else?

Reply:
No issue with that but please add a comment explaining why the commented code is there.

Reply:
Removed the commented code, added comment instead.